packages/flutter_tools/lib/src/cache.dart

```diff
@@ -356,6 +356,30 @@ class Cache {
     }
   }
 
+  String get devToolsVersion {
+    if (_devToolsVersion == null) {
+      final Directory devToolsDir = getCacheDir('dart-sdk/bin/resources/devtools');
+      if (!devToolsDir.existsSync()) {
+        throwToolExit('Could not determine DevTools version.');
```

Comment:
We're assuming this directory exists, I'm thinking that if the directory is missing this is bug on our part, and thus we should probably throw some other kind of exception that we see in crash logging?

Reply:
Is there a specific exception that will show up in crash logs? Or another place in flutter_tools that does this that I could mimc here?

Reply:
Pretty much anything other than ToolExit will show up in crash logging. You could throw a bare `Exception`, as we do elsewhere in this file, I'm not picky :)

Reply:
done.